app/views/events/show.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="container-fluid">
+	<div id="carousel" class="col-xs-12 col-sm-12 col-md-8">
+
+		<div>
+			<%=form_tag event_path do %>
+				<div class="join-btn-div">
+					<% if @user_event.present? %>
+						<%= link_to "'Unjoin' Event", event_user_event_path(@event, @user_event), method: :delete, class: 'btn btn-info join-btn' %>
+					<% else %>
+						<%= link_to "Join this Event!", event_user_events_path(@event), method: :post, class: 'btn btn-info join-btn' %>
+					<% end %>
+				</div>
+			<% end %>
+		</div>
+
+		<div id="myCarousel" class="carousel slide" data-ride="carousel">
+		  <!-- Indicators -->
+		  <ol class="carousel-indicators">
+				<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+	      <li data-target="#myCarousel" data-slide-to="1"></li>
+	      <li data-target="#myCarousel" data-slide-to="2"></li>
+		  </ol>
+
+		  <!-- Wrapper for slides -->
+		  <div class="carousel-inner" role="listbox">
+		    <div class="item active">
+		    	<%= image_tag('/uploads/litter/image/124/20170529_194751.jpg', :class => "img-responsive carousel1")%>
+				</div>
+
+		    <div class="item">
+		    	<%= image_tag('/uploads/litter/image/125/20170425_190321.jpg', :class => "img-responsive carousel1")%>
+				</div>
+
+		    <div class="item">
+					<%= image_tag('/uploads/litter/image/118/20170425_190338.jpg', :class => "img-responsive carousel1")%>
+				</div>
+		  </div>
+
+		  <!-- Left and right controls -->
+		  <a class="left carousel-control" href="#myCarousel" data-slide="prev">
+		    <span class="glyphicon glyphicon-chevron-left"></span>
+		    <span class="sr-only">Previous</span>
+		  </a>
+		  <a class="right carousel-control" href="#myCarousel" data-slide="next">
+		    <span class="glyphicon glyphicon-chevron-right"></span>
+		    <span class="sr-only">Next</span>
+		  </a>
+		</div>
+	</div>
+
+	<div class="col-xs-12 col-sm-12 col-md-4 details_and_map">
+		<div class="details">
+			<h1> Clean Up Event ! </h1>
+			<br>
+			<h4> Date: <%= @event.planned_date %> </h4>
+			<h4> Time: <%= @event.planned_time.to_time.utc.strftime("%H:%M") %> </h4>
+			<h4> Location: 2250 Woolwich Ave, Burnaby, BC V5B 4E2 </h4>       											<%# @event.location.address %>
+			<h3> Organizer of this event: <span style="font-weight:bold"> <%= @event.organizers.first.user.full_name %> </span> </h3>
+			<h4> Email organizer: <%= @event.organizers.first.user.email %> </h4>
+			<br>
+			<h4>People joined: <%= @event.users.count %> </h4>
+		</div>
+
+		<div id="map" class='map-event'></div>
+		<script>
+			var trash = {'lat': 49.263921, 'lng': -122.971534};
+
+			handler = Gmaps.build('Google');
+			handler.buildMap({ provider: {}, internal: {id: 'map'}}, function(){
+				marker = handler.addMarkers([
+					trash
+				]);
+				handler.getMap().setZoom(17);
+				handler.map.centerOn(trash);
+			});
+		</script>
+	</div>
```

Comment:
Something weird going on with the indentation here. Not sure if this is a GitHub thing or whether this is in the actual file. If it is the latter, please fix :)